### ISSUE TITLE
Copy/Paste not working on Entry Field in Windows OS

### ISSUE THREAD

**Issue by pkk-code** [fyne-io/fyne#981]
Describe the bug:

Copy/Paste from/to Entry field is not working using Ctrl+C, Ctrl+V. Though it works on Mac OS or using context menu on entry field in Windows OS.

To Reproduce:

Copy text from anywhere using Ctrl+C and paste Ctrl+V it to entry field (or)

Copy text from entry field using Ctrl+C and paste it in text editor Ctrl+V

Example code:
package main
import (
	"fyne.io/fyne"
	"fyne.io/fyne/app"
	"fyne.io/fyne/widget"
)
var form *widget.Form
func main() {
	a := app.New()
	w := a.NewWindow("Hello")
	form = &widget.Form{
		OnCancel: func() {
		},
		OnSubmit: func() {
		},
	}
	form.Append("User Name", widget.NewEntry())
	w.SetContent(form)
	w.Resize(fyne.NewSize(40, 40))
	w.ShowAndRun()
}

### Device (please complete the following information):
 - OS: Windows 
 - Version: 10
 - Go version: go version go1.14 windows/amd64
 - Fyne version: 1.2.4

**Comment by andydotxyz** [comment]
We are struggling to reproduce this as it works on our development computers.
Is it a non-enUS/GB keyboard configuration or is there anything noteworty on your setup?

**Comment by andydotxyz** [comment]
Possible duplicate of #952

**Comment by pkk-code** [comment]
Our development computers are with en-US keyboard configuration. This issue happens on all of windows computers.

**Comment by andydotxyz** [comment]
Thanks for the information.
If you run fyne_demo and go to the "Advanced" can you please press Ctrl-C and post a screenshot of the data that appears? (you may need to scroll it so it's readable sorry)

**Comment by andydotxyz** [comment]
It might be a daft question but I assume that normal keyboard input is working OK?

**Comment by pkk-code** [comment]
![image](https://user-images.githubusercontent.com/37810104/81955224-d149bf80-95d7-11ea-84bf-929fc25dd25d.png)
![image](https://user-images.githubusercontent.com/37810104/81955253-dad32780-95d7-11ea-93ea-1ea184263b29.png)

Attached the screenshot. Yes, normal keyboard input is working fine. Thanks.

**Comment by andydotxyz** [comment]
Thank you. This is very strange.
If you press "C" without Ctrl does that advanced screen report it or stay blank?

**Comment by pkk-code** [comment]
![image](https://user-images.githubusercontent.com/37810104/81965831-9dc26180-95e6-11ea-9322-1172f8e2572e.png)
It reports rune:c.

**Comment by andydotxyz** [comment]
OK, that makes sense I guess. Something in the windows driver is hiding key presses.
Maybe it is not computer specific but version specific. I will do more testing - thanks for pointing me in the right direction :)

**Comment by andydotxyz** [comment]
Hmm, I still cannot replicate it here on `1.2.4` or on `develop`.
Would it be possible for you to test the fyne_demo app on `develop` branch and see if you get the same behaviour please?

**Comment by andydotxyz** [comment]
I have a hunch here but as I cannot replicate some help would be really useful. Would it be possible to apply the following patch to `internal/driver/glfw/window.go` and test it out?
If it fixes the problem then great! and if not the debug output will help a lot thanks.

```diff
diff --git a/internal/driver/glfw/window.go b/internal/driver/glfw/window.go
index 3f531ea4..2c80c36d 100644
--- a/internal/driver/glfw/window.go
+++ b/internal/driver/glfw/window.go
@@ -5,7 +5,9 @@ import (
        "bytes"
        "image"
        _ "image/png" // for the icon
+       "log"
        "runtime"
+       "strings"
        "sync"
        "time"
 
@@ -831,11 +833,14 @@ var keyNameMap = map[string]fyne.KeyName{
 func keyToName(code glfw.Key, scancode int) fyne.KeyName {
        ret, ok := keyCodeMap[code]
        if ok {
+               log.Println("Code", ret)
                return ret
        }
 
        keyName := glfw.GetKeyName(code, scancode)
-       ret, ok = keyNameMap[keyName]
+       lower := strings.ToLower(keyName)
+       ret, ok = keyNameMap[lower]
+       log.Println("Name", keyName, lower, ret)
        if !ok {
                return ""
        }
```

**Comment by pkk-code** [comment]
> Hmm, I still cannot replicate it here on `1.2.4` or on `develop`.
> Would it be possible for you to test the fyne_demo app on `develop` branch and see if you get the same behaviour please?

I believe you solved the problem on develop branch. The key press event now shows 'C' in the advanced tab and i could copy/paste to the entry field.

Thank you for your time looking into this strange issue. May i know, if there would be patch available to fix this?

![image](https://user-images.githubusercontent.com/37810104/82054867-9ace7c00-968d-11ea-9067-a37eeb9a4a51.png)

**Comment by pkk-code** [comment]
I looked at the difference between the package i have got in go .. glfw/window.go and develop glfw/window.go and could see this difference. May be that's the clue how its working in develop branch.

![image](https://user-images.githubusercontent.com/37810104/82065958-c147e380-969c-11ea-87dc-9c34039aa078.png)

**Comment by andydotxyz** [comment]
We will be releasing these fixes in 1.3 at the beginning of June, thanks for the help tracking this down.
